docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '3.7'
+
+services:
+  minio:
+    container_name: minio
+    image: minio/minio:latest
```

Comment:
Is `latest` safe for Minio? It seems to me that Minio has had problems with upgrades in the past, and I'd hate to see the data ruined because we did `docker-compose down && docker-compose up` for some reason.

Reply:
Damn wanted to change that and forgot...